src/providers/nfsProvider.ts

```diff
@@ -10,9 +10,9 @@ import { Provider, NFSConfig } from '../common/interfaces';
 @injectable()
 export class NFSProvider implements Provider {
   public constructor(
-    @inject(SERVICES.PROVIDER_CONFIG) protected readonly config: NFSConfig,
+    protected readonly config: NFSConfig,
     @inject(SERVICES.LOGGER) protected readonly logger: Logger,
-    @inject(SERVICES.QUEUE_FILE_HANDLER) protected readonly queueFileHandler: QueueFileHandler
+    protected readonly queueFileHandler: QueueFileHandler
```

Comment:
because its dont need to be inject able

Reply:
If you decide to make a class not injectable, you need to implement it in all references. You can't just remove the injection in one file and keep it in another. You kept the inject in the containerConfig and many other files.
This will cause a mess. The code should be clean and consistent.